tools/tpm2_policyauthorize.c

```diff
@@ -16,7 +16,7 @@ struct tpm2_policyauthorize_ctx {
    //File path for the policy qualifier data
    const char *qualifier_data_path;
    //File path for the verifying public key name
-   const char *verifying_pubkey_path;
+   const char *verifying_pubkey_name_path;
```

Comment:
You should have split this up in 2 patches, I won't nack it this time, but I will next time. 1 patch for the variable rename and 1 patch for the actual logging error fix.